site/sfguides/src/how_to_resolve_data_with_fullcontact_and_snowflake/how_to_resolve_data_with_fullcontact_and_snowflake.md

```diff
@@ -240,8 +244,7 @@ By following this guide you learned how to use the FullContact for Snowflake app
 
```

Comment:
@danhlee Rename the last step to "Conclusion And Resources" and make the layout/format similar to https://quickstarts.snowflake.com/guide/getting_started_with_snowflake_arctic/index.html#5